src/resolvers-cassandra/Edges/queries.js

```diff
@@ -70,45 +41,186 @@ function locations(args, res) { // eslint-disable-line no-unused-vars
   return new Promise((resolve, reject) => {
     if (!args || !args.site) return reject('No site specified for which to lookup locations');
 
-    const query = makeSiteBboxQuery(args);
-    cassandraConnector.executeQuery(query.query, query.params)
+    const query = `
+    SELECT geofence
+    FROM fortis.sitesettings
+    WHERE sitename = ?
+    `.trim();
+
+    const params = [
+      args.sitename
+    ];
+
+    cassandraConnector.executeQuery(query, params)
     .then(rows => {
       if (!rows || !rows.length) return reject(`No geofence configured for site ${args.site}`);
       if (rows.length > 1) return reject(`More than one geofence configured for site ${args.site}`);
       if (!rows[0].geofence || rows[0].geofence.length !== 4) return reject(`Bad geofence for site ${args.site}`);
 
       const bbox = rows[0].geofence;
-      featureServiceClient.fetchByBbox({north: bbox[0], west: bbox[1], south: bbox[2], east: bbox[3]})
-      .then(locations => {
-        resolve({
-          edges: locations.map(location => ({name: location.name, coordinates: location.bbox}))
-        });
-      })
-      .catch(reject);
+      return featureServiceClient.fetchByBbox({north: bbox[0], west: bbox[1], south: bbox[2], east: bbox[3]});
+    })
+    .then(locations => {
+      resolve({
+        edges: locations.map(location => ({name: location.name, coordinates: location.bbox}))
+      });
     })
     .catch(reject);
   });
 }
 
 /**
- * @param {{site: string, timespan: string, sourceFilter: string[]}} args
+ * @param {{site: string, timespan: string, sourceFilter: string[], mainEdge: string, originalSource: string}} args
  * @returns {Promise.<{runTime: string, edges: Array<{name: string, mentions: number, coordinates: number[], population: number}>}>}
  */
 function popularLocations(args, res) { // eslint-disable-line no-unused-vars
+  return new Promise((resolve, reject) => {
+    const { period, periodType } = parseTimespan(args.timespan);
+
+    const query = `
+    SELECT placename, placeid, mentioncount
+    FROM fortis.popularplaces
+    WHERE period = ?
+    AND periodtype = ?
+    AND pipelinekey = ?
+    AND externalsourceid = ?
+    AND conjunctiontopic1 = ?
```

Comment:
This is one for @erikschlegel. I don't particularly care about the naming here because we have quite a few things in the database that require documentation already.